### DISCUSSION TITLE
增加断点续传功能，传输错误文件再次传输功能

### DISCUSSION THREAD

**Discussion by chgo-d** [localsend/localsend#940] [L-简体中文]
希望能增加按钮，或者自动对出错的文件进行再次尝试。如果发多个文件，中途网络断开，又恢复，断开时正在传输的文件会报错误，恢复后其余文件可以继续传输完成，但是错误文件无法再次传输，需要重新添加，再次发起。在传输完成时，如果有文件传输错误，希望有按钮可以直接把错误文件再次传输

**Comment by switchYello** [comment]
同，部分文件失败部分成功只能重新选择，有点麻烦
